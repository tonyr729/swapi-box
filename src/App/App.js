import React, { Component } from 'react';
import Intro from './Components/Statefull/Intro/index'
import Main from './Components/Statefull/Main/index'
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
      </div>
    );
  }
}

export default App;
