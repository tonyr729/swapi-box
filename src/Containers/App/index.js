import React, { Component } from 'react';
import Crawl from '../../Components/Statefull/Crawl/index';
import Main from '../../Containers/Main/index';
import './index.css';

class App extends Component {
  render() {
    return (
      <div className="App">
      <Main />
      <Crawl />
      </div>
    );
  }
}

export default App;
