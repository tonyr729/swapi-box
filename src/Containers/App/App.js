import React, { Component } from 'react';
import Crawl from '../../Components/Statefull/Crawl/index'
import Main from '../../Containers/Main/index'
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
      <Main />
      {/* <Crawl /> */}
      </div>
    );
  }
}

export default App;
