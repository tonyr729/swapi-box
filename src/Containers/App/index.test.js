import React from 'react';
import ReactDOM from 'react-dom';
import App from './index';
import renderer from 'react-test-renderer';
import APIManager from '../../Helpers/API/API Manager/';
jest.mock('../../Helpers/API/API Manager/', () => {
  return jest.fn().mockImplementation(() => {
    return {
      fetchCrawl: jest.fn().mockImplementation(() => Promise.resolve({name: 'tony'})),
    };
  });
});

it('should match snapshot', () => {
  const app = renderer.create(<App />).toJSON();

  expect(app).toMatchSnapshot();
});
