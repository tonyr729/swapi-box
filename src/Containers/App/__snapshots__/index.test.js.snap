// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="App"
>
  <div
    className="main"
  >
    <div
      className="header"
    >
      <h1
        className="header-title"
      >
        SWAPI-Box
      </h1>
      <div
        className="favorite-div"
        onClick={[Function]}
      >
        <p
          className="favorite-text"
        >
          Favorites
        </p>
        <div
          className="favorite-count"
        >
          <p>
            0
          </p>
        </div>
      </div>
      <div
        className="button-container"
      >
        <button
          onClick={[Function]}
        >
          People
        </button>
        <button
          onClick={[Function]}
        >
          Planets
        </button>
        <button
          onClick={[Function]}
        >
          Vehicles
        </button>
      </div>
    </div>
    <div
      className="card-container"
    />
  </div>
  <div
    className="crawl-container"
  >
    <div
      className="fade"
    />
    <section
      className="star-wars"
    >
      <div
        className="crawl"
      >
        <div
          className="title"
        >
          <p>
            Episode 
            
          </p>
          <h1>
            
          </h1>
        </div>
        <p>
          
        </p>
        <p>
          
        </p>
      </div>
    </section>
  </div>
</div>
`;
